### PR TITLE
Updated Italian

### PR DESCRIPTION
**Pull request by Raffox97**
* Added missing strings
* Edited some zone names
* Removed unused strings

### CODE REVIEW

File: web/locales/it.json [hunk 1]
```diff
@@ -420,17 +421,17 @@
...
         "CL-SEM": {
             "countryName": "Cile",
-            "zoneName": "SEM"
+            "zoneName": "Sistema de Magallanes"
         },
-        "CL-SEA": {
+        "CL-SEN": {
             "countryName": "Cile",
-            "zoneName": "SEA"
+            "zoneName": "Sistema Eléctrico Nacional"
```
Comment (fbarl) [inline web/locales/it.json:434]: Could you please provide a source for these 3 zone names? If that's how these zones are indeed called, we might want to re-use these zone names across all other translations. cc @robertahunt

Comment (Raffox97) [reply web/locales/it.json:434]: Sure, there is a page on the Spanish wikipedia. Here's the link: https://es.wikipedia.org/wiki/Interconexi%C3%B3n_el%C3%A9ctrica_de_Chile :)

Comment (fbarl) [reply web/locales/it.json:434]: Haha ok good point, the second sentence already explains it all:

> Hasta 2017 existían cuatro sistemas intercontectados que operaban aisladamente de los otros, pero ese año se unieron los dos principales –el Sistema Interconectado Central (SIC) y el Sistema Interconectado del Norte Grande (SING)– en el Sistema Eléctrico Nacional (SEN). Al SEN se le suman el Sistema de Aysén (SEA) y el Sistema de Magallanes (SEM), que operan en la zona austral del país.

Could you please update the english locale `en.json` with the same zone names while we're at it? Thanks!

Comment (Raffox97) [reply web/locales/it.json:434]: Done :)

**Pull request merged**